src/pages/Game.js

```diff
@@ -29,12 +35,16 @@ export const Game = () => {
     <div className='App'>
       <header>
         <h1>Tic-Tac-Toe</h1>
-        <div className='user'>
-          <Button variant='contained' size='small'>
-            Change
-          </Button>
-        </div>
       </header>
+      <br />
+      <div className='finishedGame'>
+        {isFinished && winner != null ? (
+          <Alert severity='success'>{winner === info.players[0].id ? info.players[0].username : info.players[1].username} won! 🎉</Alert>
+        ) : (
+          ''
+        )}
+        {isFinished && winner == null ? <Alert severity='info'>THERE IS NO WINNER</Alert> : ''}
```

Comment:
Again here we can do:
`{isFinished && winner !== null && <Alert... >}`.
And also we could set a more descriptive text like: Game is a draw, or Stalemate.